### PR TITLE
Add pixelpulsecraft

### PR DESCRIPTION
**Pull request by elxAl33**
## Type of change

- [x] Add new directory for a server
- [x] Update directory of a server
- [x] Remove directory of a server
- [x] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

## Further comments
background
background@2x
icon
icon@2x
[manifest.json](https://github.com/LabyMod/server-media/files/14073590/manifest.json)

### CODE REVIEW

File: minecraft_servers/PixelPulseCraft
Comment (maaaathis) [inline minecraft_servers/PixelPulseCraft:1]: Remove this file out of your branch.

File: minecraft_servers/background@2x.png
Comment (maaaathis) [inline minecraft_servers/background@2x.png:1]: Compress

File: minecraft_servers/manifest.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    "%.PixelPulseCraft.de"
+  ],
+  "supported_languages": [
+    "de"
+  ],
+  "social": {
+    "twitter": "PixelPulseCraft",
+    "tiktok": "pixelpulsecraft.de",
+    "youtube": "https://www.youtube.com/channel/UCdIaEvVXlF9PtOiabh2UjBA",
+    "instagram": "labymod",
```
Comment (maaaathis) [inline minecraft_servers/manifest.json:15]: That doesn't look like your Instagram account. Please change or remove.

**Comment by elxAl33** [comment]
pixelpulsecraft

File: minecraft_servers/manifest.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    "twitch": "kirex94"
```
Comment (maaaathis) [inline minecraft_servers/manifest.json:16]: That doesn't look like the Twitch account of your server "PixelPulseCraft".

**Review by maaaathis — CHANGES_REQUESTED** [review]

**Comment by maaaathis** [comment]
duplicate of https://github.com/LabyMod/server-media/pull/925

**Comment by maaaathis** [comment]
Invalid background size! Size is 1366x697px, must be 1280x720px: minecraft_servers/background.png
Error: Invalid hDPI background size! Size is 1366x697px, must be 1920x1080px: minecraft_servers/background@2x.png


Error: Could not read image file: minecraft_servers/icon@2x.png
Error: Invalid icon size! Size is 64x64px, must be 2[5](https://github.com/LabyMod/server-media/actions/runs/7680192239/job/20931922436?pr=926#step:4:6)[6](https://github.com/LabyMod/server-media/actions/runs/7680192239/job/20931922436?pr=926#step:4:7)x256px: minecraft_servers/icon.png

**Comment by maaaathis** [comment]
You can use https://www.iloveimg.com or similar to resize and compress the images. Only upscaling would be a problem, but I assume you have the original images in larger resolutions.

**Comment by elxAl33** [comment]
> You can use https://www.iloveimg.com or similar to resize and compress the images. Only upscaling would be a problem, but I assume you have the original images in larger resolutions.

Okay i did what i need to do now?

**Comment by elxAl33** [comment]
> You can use https://www.iloveimg.com or similar to resize and compress the images. Only upscaling would be a problem, but I assume you have the original images in larger resolutions.

I did what i do now?

**Comment by maaaathis** [comment]
> > You can use https://www.iloveimg.com or similar to resize and compress the images. Only upscaling would be a problem, but I assume you have the original images in larger resolutions.
> 
> I did what i do now?

Create here a directory named "pixelpulsecraft" and move your previous pushed files to this directory: https://github.com/elxAl33/server-media/tree/add-pixelpulsecraft/minecraft_servers. I didn't realize before that you had pushed your files in the main directory, sorry.

Then upload there also your new and compressed images. The PR updates automatically.

**Comment by elxAl33** [comment]
> > > You can use https://www.iloveimg.com or similar to resize and compress the images. Only upscaling would be a problem, but I assume you have the original images in larger resolutions.
> > 
> > 
> > I did what i do now?
> 
> Create here a directory named "pixelpulsecraft" and move your previous pushed files to this directory: https://github.com/elxAl33/server-media/tree/add-pixelpulsecraft/minecraft_servers. I didn't realize before that you had pushed your files in the main directory, sorry.
> 
> Then upload there also your new and compressed images. The PR updates automatically.

Ik im dumb but how and where i create the drectory and how i move it can you maybe make a vid or show me with screen shots?
Sorry if im to complicated

**Comment by maaaathis** [comment]
> > > > You can use https://www.iloveimg.com or similar to resize and compress the images. Only upscaling would be a problem, but I assume you have the original images in larger resolutions.
> > > 
> > > 
> > > I did what i do now?
> > 
> > 
> > Create here a directory named "pixelpulsecraft" and move your previous pushed files to this directory: https://github.com/elxAl33/server-media/tree/add-pixelpulsecraft/minecraft_servers. I didn't realize before that you had pushed your files in the main directory, sorry.
> > Then upload there also your new and compressed images. The PR updates automatically.
> 
> Ik im dumb but how and where i create the drectory and how i move it can you maybe make a vid or show me with screen shots? Sorry if im to complicated

On the "New File" wizard here at github.com, you can simply choose `pixelpulsecraft/manifest.json` as the file name to create the `manifest.json` file inside the `pixelpulsecraft` directory. Then you can simply upload the other files to this directory because it will already exist.

**Comment by elxAl33** [comment]
> > > > > You can use https://www.iloveimg.com or similar to resize and compress the images. Only upscaling would be a problem, but I assume you have the original images in larger resolutions.
> > > > 
> > > > 
> > > > I did what i do now?
> > > 
> > > 
> > > Create here a directory named "pixelpulsecraft" and move your previous pushed files to this directory: https://github.com/elxAl33/server-media/tree/add-pixelpulsecraft/minecraft_servers. I didn't realize before that you had pushed your files in the main directory, sorry.
> > > Then upload there also your new and compressed images. The PR updates automatically.
> > 
> > 
> > Ik im dumb but how and where i create the drectory and how i move it can you maybe make a vid or show me with screen shots? Sorry if im to complicated
> 
> On the "New File" wizard here at github.com, you can simply choose `pixelpulsecraft/manifest.json` as the file name to create the `manifest.json` file inside the (new) `pixelpulsecraft` directory. Then you can simply upload the other files to this directory because it will already exist.

I hope i did it now right [https://github.com/elxAl33/server-media/pull/1](url)

**Comment by maaaathis** [comment]
> > > > > > You can use https://www.iloveimg.com or similar to resize and compress the images. Only upscaling would be a problem, but I assume you have the original images in larger resolutions.
> > > > > 
> > > > > 
> > > > > I did what i do now?
> > > > 
> > > > 
> > > > Create here a directory named "pixelpulsecraft" and move your previous pushed files to this directory: https://github.com/elxAl33/server-media/tree/add-pixelpulsecraft/minecraft_servers. I didn't realize before that you had pushed your files in the main directory, sorry.
> > > > Then upload there also your new and compressed images. The PR updates automatically.
> > > 
> > > 
> > > Ik im dumb but how and where i create the drectory and how i move it can you maybe make a vid or show me with screen shots? Sorry if im to complicated
> > 
> > 
> > On the "New File" wizard here at github.com, you can simply choose `pixelpulsecraft/manifest.json` as the file name to create the `manifest.json` file inside the (new) `pixelpulsecraft` directory. Then you can simply upload the other files to this directory because it will already exist.
> 
> I hope i did it now right [https://github.com/elxAl33/server-media/pull/1](url)

You make that chane in a new branch, so this PR won't update. Also you missed the `.json` file ending.

**Comment by elxAl33** [comment]
> Invalid background size! Size is 1366x697px, must be 1280x720px: minecraft_servers/background.png Invalid hDPI background size! Size is 1366x697px, must be 1920x1080px: minecraft_servers/[background@2x.png](mailto:background@2x.png)
> 
> Could not read image file: minecraft_servers/[icon@2x.png](mailto:icon@2x.png) Invalid icon size! Size is 64x64px, must be 2[5](https://github.com/LabyMod/server-media/actions/runs/7680192239/job/20931922436?pr=926#step:4:6)[6](https://github.com/LabyMod/server-media/actions/runs/7680192239/job/20931922436?pr=926#step:4:7)x256px: minecraft_servers/icon.png

Please Tell me i did it

**Comment by maaaathis** [comment]
> > Invalid background size! Size is 1366x697px, must be 1280x720px: minecraft_servers/background.png Invalid hDPI background size! Size is 1366x697px, must be 1920x1080px: minecraft_servers/[background@2x.png](mailto:background@2x.png)
> > Could not read image file: minecraft_servers/[icon@2x.png](mailto:icon@2x.png) Invalid icon size! Size is 64x64px, must be 2[5](https://github.com/LabyMod/server-media/actions/runs/7680192239/job/20931922436?pr=926#step:4:6)[6](https://github.com/LabyMod/server-media/actions/runs/7680192239/job/20931922436?pr=926#step:4:7)x256px: minecraft_servers/icon.png
> 
> Please Tell me i did it

I described it in https://github.com/LabyMod/server-media/pull/926#issuecomment-1913285629 and https://github.com/LabyMod/server-media/pull/926#issuecomment-1913288177.
You can also see https://stackoverflow.com/a/63635965/13061979 for further help.

**Comment by elxAl33** [comment]
> > > Invalid background size! Size is 1366x697px, must be 1280x720px: minecraft_servers/background.png Invalid hDPI background size! Size is 1366x697px, must be 1920x1080px: minecraft_servers/[background@2x.png](mailto:background@2x.png)
> > > Could not read image file: minecraft_servers/[icon@2x.png](mailto:icon@2x.png) Invalid icon size! Size is 64x64px, must be 2[5](https://github.com/LabyMod/server-media/actions/runs/7680192239/job/20931922436?pr=926#step:4:6)[6](https://github.com/LabyMod/server-media/actions/runs/7680192239/job/20931922436?pr=926#step:4:7)x256px: minecraft_servers/icon.png
> > 
> > 
> > Please Tell me i did it
> 
> I described it in [#926 (comment)](https://github.com/LabyMod/server-media/pull/926#issuecomment-1913285629) and [#926 (comment)](https://github.com/LabyMod/server-media/pull/926#issuecomment-1913288177). You can also see https://stackoverflow.com/a/63635965/13061979 for further help.

Can you Please add me on discord its the easier way to understand couse i think im dumb for it user is ayoal33_0000

**Pull request closed without merging**